tools/integration_tests/gzip/helpers/helpers.go

```diff
@@ -31,20 +30,9 @@ import (
 
 const (
 	TempFileStrLine = "This is a test file"
-	TmpDirectory    = "/tmp"
 )
 
-// Creates a temporary file (name-collision-safe) in /tmp with given content size in bytes.
-// If gzipCompress is true, output file is a gzip-compressed file.
-// contentSize is the size of the uncompressed content. In case gzipCompress is true, the actual output file size will be
-// different from contentSize (typically gzip-compressed file size < contentSize).
-// Caller is responsible for deleting the created file when done using it.
-// Failure cases:
-// 1. contentSize <= 0
-// 2. os.CreateTemp() returned error or nil handle
-// 3. gzip.NewWriter() returned nil handle
-// 4. Failed to write the content to the created temp file
-func CreateLocalTempFile(contentSize int, gzipCompress bool) (string, error) {
+func CreateDataOfSize(contentSize int) (string, error) {
```

Comment:
This function is only used within this package (gzip), so it doesn't need to be public, so it should start with small letter. Yes, it was wrong even earlier I think.

```suggestion
func createDataOfSize(contentSize int) (string, error) {
```

Reply:
If it needs to be public, then it must have heaader.

```suggestion
// CreateDataOfSize ...
func CreateDataOfSize(contentSize int) (string, error) {
```

Reply:
This is used in both packages helpers and gzip_test.

Reply:
Got it will move both methods CreateDataOfSize and DownloadGzipGcsObjectAsCompressed to gzip_test